doc/installing/cluster.md

```diff
@@ -37,40 +112,41 @@ oq_distribute = zmq
 
 [dbserver]
 multi_user = true
-file = /var/lib/openquake/db.sqlite3
+file = /var/lib/openquake/oqdata/db.sqlite3
 # daemon bind address; must be a valid IP address
-listen = 0.0.0.0
+listen = < IP address of master>
 # address of the dbserver; can be an hostname too
 # on multi-node cluster it must be the IP or hostname
 # of the master node (on the master node cfg too)
-host = w.x.y.z
-port = 1907
-receiver_ports = 1912-1920
+host = < IP address of master>
+port = 1908
+receiver_ports = 1921-1930
```

Comment:
Why not keeping 1912-1920 as it was?

Reply:
I use the same value of our installation in GEM, think they are the defaults

Reply:
Probably since we have two clusters in one we use 1912-1920 and in the other 1921-1930 but in case of a single case we should keep the range of ports continuous from 1908 to 1920.

Reply:
Update range to 1912-1920